applications/EmpireApplication/python_scripts/empire_wrapper.py

```diff
@@ -301,7 +355,7 @@ def _set_mesh(self, model_part, num_nodes, num_elements, node_coords, node_IDs,
                     name_element = "Element2D2N"
                 elif num_nodes_element == 3:
                     name_element = "Element2D3N"
-                elif num_nodes_element == 4: # TODO how to distinguish from Tetras?
+                elif num_nodes_element == 4: # TODO how to distinguish from Tetras? => use DOMAIN_SIZE of ModelPart
```

Comment:
not really .. what about quads on a 3d surface ? there are 4 nodes and  DOMAIN_SIZE = 3.
Also why will there be Tets on surface mesh ?